pom.xml

```diff
@@ -226,8 +227,31 @@
 			</activation>
 			<properties>
 				<!-- SWT on Mac OS needs to be started on main thread -->
-				<tycho.testArgLine>-XstartOnFirstThread</tycho.testArgLine>
+				<tycho.testArgLine>-XstartOnFirstThread ${tychoDefaultTestArgLine}</tycho.testArgLine>
 			</properties>
+			<build>
+				<pluginManagement>
+					<plugins>
+						<plugin>
+							<groupId>org.eclipse.tycho</groupId>
+							<artifactId>tycho-surefire-plugin</artifactId>
+							<version>${tycho-version}</version>
+							<configuration>
+								<printBundles>true</printBundles>
```

Comment:
Do you think that is necessary for the every-day use? Looks like a debug-option to me.

Reply:
Agreed, I removed now.